helm/ingress-controller/templates/credentials-secret.yaml

```diff
@@ -1,10 +1,12 @@
-{{- if or (not (empty .Values.apiKey)) (not (empty .Values.authtoken)) }}
+{{- if eq .Values.credentials.secret.create true }}
+{{- if or (not (empty .Values.credentials.apiKey)) (not (empty .Values.credentials.authtoken)) }}
```

Comment:
If both of these are required when creating the secret, I think its fine to remove this line. Otherwise, if you set `secrete.create=true` and leave both `apiKey` and `authToken` empty, you wouldn't hit the `required` checks.

Reply:
ahh thanks, yeah they should both be required if the create is set to true. Will remove this